docs/explorer.rst

```diff
@@ -255,16 +255,16 @@ the ``has`` filter to find cases that have ``4d-seismic`` data:
 
 .. code-block:: python
 
-    from fmu.sumo.explorer import Explorer, Filters
+    from fmu.sumo.explorer import Explorer, filters
```

Comment:
Why is `Filters` changed to `filters` here?

Reply:
I guess the convention is lowercase filenames? Which `Filters.py` is breaking.

Should `Filters.py` be renamed to `filters.py` as well?

Reply:
Oops. I thought I had renamed the file to filters.py.